diff --git a/components/hal/mmu_hal.c b/components/hal/mmu_hal.c
index 95441f7463e..5bd051e0030 100644
--- a/components/hal/mmu_hal.c
+++ b/components/hal/mmu_hal.c
@@ -41,7 +41,8 @@ void mmu_hal_unmap_all(void)
 {
 #if SOC_MMU_PER_EXT_MEM_TARGET
     mmu_ll_unmap_all(MMU_LL_FLASH_MMU_ID);
-    mmu_ll_unmap_all(MMU_LL_PSRAM_MMU_ID);
+    /* SILICONLOOP EXPERIMENT: PSRAM MSPI 控制器未初始化时写其 MMU 寄存器会触发 PMP fault */
+    /* mmu_ll_unmap_all(MMU_LL_PSRAM_MMU_ID); */
 #else
     for (int i = 0; i < s_ctx.core_nums; i++) {
         mmu_ll_unmap_all(i);
